crates/solvers/Cargo.toml

```diff
@@ -36,7 +36,7 @@ thiserror = "1"
 tokio = { version = "1", features = ["macros", "rt-multi-thread", "signal", "time"] }
 toml = "0.7"
 tower = "0.4"
-tower-http = { version = "0.4", features = ["trace"] }
+tower-http = { version = "0.4", features = ["limit", "trace"] }
```

Comment:
slightly unrelated trivial change needed to make the solver binary compile in isolation.